docs/en/File-System-API.md

```diff
@@ -63,74 +65,62 @@ FileOutStream out = tfs.getOutputStream(path, options);
 Tachyon uses two different storage types: Tachyon managed storage and under storage. Tachyon managed
 storage is the memory, SSD, and/or HDD allocated to Tachyon workers. Under storage is the storage
 resource managed by the underlying storage system, such as S3, Swift or HDFS. Users can specify the
-interaction with the Tachyon's native storage and under storage through the `TachyonStorageType` and
-`UnderStorageType` respectively. Note that read operations are affected by `TachyonStorageType`
-(the data can be stored in more Tachyon workers) but not `UnderStorageType`.
+interaction with the Tachyon's native storage and under storage through `ReadType` and `WriteType`.
+Both these enums are responsible for controlling the behavior of any write operations. In the case
+of `ReadType`, it specifies whether the data should be written concurrently to Tachyon storage.
 
-Below is a table of commonly used `TachyonStorageType` and `UnderStorageType` combinations as well
-as the deprecated `WriteType, ReadType` equivalents.
+Below is a table of the expected behaviors of `ReadType`.
 
 <table class="table table-striped">
-<tr><th>TachyonStorageType</th><th>UnderStorageType</th><th>Previously</th>
-<th>Read Behavior</th><th>Write Behavior</th>
+<tr><th>Read Type</th><th>Behavior</th>
 </tr>
 <tr>
-  <td>PROMOTE</td>
-  <td>NO_PERSIST</td>
-  <td>MUST_CACHE, CACHE_PROMOTE</td>
+  <td>CACHE_PROMOTE</td>
   <td>Data is moved to the highest tier in the worker where the data was read. If the data was not
   in the Tachyon storage of the local worker, a replica will be added to the local Tachyon worker
-  for each completely read data block.</td>
-  <td>Data is written synchronously to a Tachyon worker.</td>
+  for each completely read data block. This is the default read type.</td>
 </tr>
 <tr>
-  <td>PROMOTE</td>
-  <td>SYNC_PERSIST</td>
-  <td>CACHE_THROUGH, CACHE_PROMOTE</td>
-  <td>Data is moved to the highest tier in the worker where the data was read. If the data was not
-  in the Tachyon storage of the local worker, a replica will be added to the local Tachyon worker
-  for each completely read data block.</td>
-  <td>Data is written synchronously to a Tachyon worker and the under storage system.</td>
-</tr>
-<tr>
-  <td>STORE</td>
-  <td>NO_PERSIST</td>
-  <td>MUST_CACHE, CACHE</td>
+  <td>CACHE</td>
   <td>If the data was not in the Tachyon storage of the local worker, a replica will be added to the
   local Tachyon worker for each completely read data block.</td>
-  <td>Data is written synchronously to a Tachyon worker. Data is not written to the under storage
-  system.</td>
 </tr>
 <tr>
-  <td>STORE</td>
-  <td>SYNC_PERSIST</td>
-  <td>CACHE_THROUGH, CACHE</td>
-  <td>If the data was not in the Tachyon storage of the local worker, a replica will be added to the
-  local Tachyon worker for each completely read data block.</td>
+  <td>NO_CACHE</td>
+  <td>Data is read preferrably from Tachyon storage, then under storage. No replicas will be
```

Comment:
Maybe then we can say "Like CACHE, but no replicas will be added to Tachyon storage".